doc/importexport.rst

```diff
@@ -231,6 +231,28 @@ optimum values that will produce an acceptable mesh.
 
     exporters.export(result, '/path/to/file/mesh.vrml', tolerance=0.01, angularTolerance=0.1)
 
+Exporting DXF
+##############
+
+By default, the DXF exporter will output splines exactly as they are represented by the OpenCascade kernel. Unfortunately some software can not handle higher-order splines, so if some of curves and arcs appear to be missing CadQuery can be instructed to approximate splines during the export process. This approximation is controlled by the following options:
+
+* ``approx`` - ``None``, ``"spline"`` or ``"arc"``. ``"spline"`` results in all splines approximated with cubic splines. ``"arc"`` results in all curves approximated with arcs and line segments.
+* ``tolerance``: Acceptable error of the approximation, in the DXF's coordinate system. Defaults to 0.001 (1 thou for inch-scale drawings, 1 µm for mm-scale drawings).
+
+.. code-block:: python
+
+    import cadquery as cq
+    from cadquery import exporters
+
+    result = cq.Workplane().box(10, 10, 10)
+
+    exporters.exportDXF(
```

Comment:
```suggestion

    cq.exporters.exportDXF(
```

box is not a good example here.  I think an example of creating a Workplane result object can be omitted  - just show the usage with approx.